### DISCUSSION TITLE
In-browser PostgreSQL playground

### DISCUSSION THREAD

**Discussion by nalgeon** [electric-sql/pglite#21] [Show and tell]
Hey folks, thanks for this great project! I've created an embeddable in-browser [playground](https://codapi.org/postgres-pglite/) on top of it. The project is also open source, so if you need a playground somewhere (maybe in the docs), feel free to use it. Just FYI.

**Comment by nicognaW** [comment]
Excellent effort from both pglite and OP's codpai playground! I also hacked a Prisma Studio-style basic table editor [here](https://nicognaw.github.io/pglite-table-editor/) (code available [here](https://github.com/nicognaW/pglite-table-editor)), and I hope it can help anyone else who wants the same thing I did.

**Comment by JfrAziz** [comment]
I am also create similiar project with more features like multiple DB, ERD generator, and query history. check it [here](https://pg.datawan.id/) and [code](https://github.com/datawan-labs/pg) 

![image](https://github.com/electric-sql/pglite/assets/44570698/42a87bc1-0871-4b89-a5e4-b2cf298868ae)

![image](https://github.com/electric-sql/pglite/assets/44570698/5070f9bc-161b-4e71-84c5-fb010d12b4c9)

**Reply by samwillis** [reply]
That's incredible!

What plans do you have for it?

**Reply by JfrAziz** [reply]
Thanks, this library is great 🚀 🚀. 

I do not have plan yet atm, may be add more features and waiting for another postgres extensions.
